tutorials/viewports/custom_postprocessing.rst

```diff
@@ -1,81 +1,74 @@
 .. _doc_custom_postprocessing:
 
-Custom post-processing 
+Custom post-processing
 ======================
 
 Introduction
 ------------
 
-Godot provides many post-processing effects out of the box including, Bloom, DOF, and SSAO.
-Sometimes you will want to write your own post-processing effect. This can be done easily
-in Godot by rendering your scene into a :ref:`Viewport <class_Viewport>` and then rendering the 
-:ref:`Viewport's <class_Viewport>` :ref:`texture <class_ViewportTexture>` to a full screen quad. 
+Godot provides many post-processing effects out of the box including, Bloom, DOF, and SSAO. Sometimes you
+want to write your own custom effect. Here's how you can do so.
 
-.. note:: At the time of writing Godot does not support rendering to multiple buffers at the same 
-          time so the post-processing shader will not have access to normals, or anything else. 
-          You only have access to the fullscreen color texture. 
+Post-processing effects are shaders applied to a frame after Godot rendered it. You first want to render
+your scene into a :ref:`Viewport <class_Viewport>`, then render the ``Viewport``
+inside a :ref:`ViewportTexture <class_ViewportTexture>` and show it on the screen.
 
-Screen reading shaders
-----------------------
+The easiest way to implement a custom post-processing shader is to use Godot's built-in ability to read from
+the screen texture. If you're not familiar with this, you should read the :ref:`Screen Reading Shaders
+Tutorial <doc_screen-reading_shaders>` first.
 
-Before starting, a brief note on screen reading shaders is in order.
+.. note:: At the time of writing Godot does not support rendering to multiple buffers at the same time. Your
```

Comment:
The rest of the notes on this page start the text on its own line. Can you do the same here, for consistency?